spec/src/main/asciidoc/servlet-spec-body.adoc

```diff
@@ -1387,9 +1370,9 @@ methods of the `ServletRequest` interface:
 Only one attribute value may be associated
 with an attribute name.
 
-Attribute names beginning with the prefixes of
-`java.` and `javax.` are reserved for definition by this specification.
-Similarly, attribute names beginning with the prefixes of `sun.` ,
+Attribute names beginning with the prefix of
+`jakarta.` are reserved for definition by this specification.
+Similarly, attribute names beginning with the prefixes of `java.`, `sun.` ,
 `com.sun., oracle and com.oracle` are reserved for definition by Oracle
```

Comment:
Do we still need this bit? I don't know if we should still reserve the Sun and Oracle namespaces.

Reply:
I'd say no.

Reply:
I don't think we should say anything about `java.`, `sun.`, `com.sun,`, `oracle` and `com.oracle`.

However, I think it may be a frequent porting mistake to forget to change the `javax.` names (for those not using the constants).   We could do an IAE to catch gets of the old names?  Or maybe a note to say that `Containers may throw IllegalArgumentException for unsupported attributes (eg. old javax. attributes` ?

Reply:
I would prefer to say nothing, so implementation could add their own backwards compatibility handling.

Reply:
Thanks for the discussion. Indeed since we are going to [empty also the change-log](https://www.eclipse.org/lists/servlet-dev/msg00183.html), it will be confusing to mention `javax. *` which refer to the past mostly.